### ISSUE TITLE
Terminal text is blur

### ISSUE THREAD

**Issue by kevinah95** [Microsoft/vscode#35988]
- VSCode Version: 1.17.0
- OS Version: macOS Sierra 10.12.6

Steps to Reproduce:

1. Run some routine into vscode terminal. Notice the terminal text is blur in bottom area:

![image](https://user-images.githubusercontent.com/7193395/31392156-c1ae903c-ad95-11e7-8985-210e0f90a634.png)
 
2. When maximize the window, terminal re-render the text and fix the blurring text:

![image](https://user-images.githubusercontent.com/7193395/31392359-4d796768-ad96-11e7-9707-770aa0834e9a.png)

3. I use [Nord Theme](https://github.com/arcticicestudio/nord-visual-studio-code) but disabled and produced the same blur effect:

![image](https://user-images.githubusercontent.com/7193395/31392698-2e742258-ad97-11e7-8e70-26c0461b7dbb.png)
 

Reproduces without extensions: Yes/No

**Comment by Tyriar** [comment]
I believe this is tracked in https://github.com/sourcelair/xterm.js/issues/985

Are you at window.zoomLevel: 1 by chance?

**Comment by kevinah95** [comment]
I use this setting:
```
{
    "vsicons.projectDetection.autoReload": true,
    "workbench.iconTheme": "vscode-icons",
    "editor.fontLigatures": true,
    "editor.fontFamily": "'Fira Code'",
    "editor.fontSize": 12,
    "editor.fontLigatures": true,
    "window.zoomLevel": -1,
    "todohighlight.isEnable": false,
    "workbench.colorTheme": "Nord",
    "typescript.check.npmIsInstalled": false
}
```

But with your suggestion the blur disappear

**Comment by Tyriar** [comment]
Yeah setting zoomLevel to 0 should fix this. Will keep this open until 
https://github.com/sourcelair/xterm.js/issues/985 is fixed.

**Comment by varadero** [comment]
I have similar problem though it is more visible on my laptop monitor. I just upgraded from 1.17 to 1.17.1 - everything was OK before the update. Before the update, the terminal window had the same quality as the code window above:

![image](https://user-images.githubusercontent.com/7610713/31407989-a1224672-ae0f-11e7-9bdf-ab5e2b054899.png)

**Comment by ivancuric** [comment]
+1, this is using Input Mono @14px:
![image](https://user-images.githubusercontent.com/2827047/31430479-427aba0c-ae71-11e7-9bc2-9da02bfb5b97.png)

**Comment by Tyriar** [comment]
For the issue in 1.17.1 when window.zoomLevel is 0, that is tracked in https://github.com/Microsoft/vscode/issues/35991

**Comment by ItsNoHax** [comment]
Oh thank god, thought I was tripping when my terminal text was looking different, I can confirm I use zoomLevel -1 too.

**Comment by lijunray** [comment]
With zoomLevel of 0, which I've been using, it's still blurry but better than -1.

**Comment by coston** [comment]
Mine is still blurry at zoomLevel 0

**Comment by ivan-ha** [comment]
Duplicated and fixed, see #35991

**Comment by Tyriar** [comment]
Going to close this off as a duplicate of https://github.com/Microsoft/vscode/issues/35991 as I believe that's what the OP was experiencing.

https://github.com/xtermjs/xterm.js/issues/985 is still being tracked upstream which sees some rows be blurry at certain zoom levels.
